NuGet.Services.Dashboard.Operations/Infrastructure/ConfigurationProcessor.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Configuration;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NuGet
+{
+    public class ConfigurationProcessor
+    {
+        private ISecretReaderFactory _secretReaderFactory;
+
+        public ConfigurationProcessor(ISecretReaderFactory secretReaderFactory)
+        {
+            _secretReaderFactory = secretReaderFactory;
+        }
+
+        public void InjectSecretsIntoConfiguration()
```

Comment:
Nifty! I'm just wondering if we can pass the ConfigurationManager.AppSettings to this method and use the argument? That makes the code a bit more readable (e.g. where are values being injected? `InjectSecretsInto(NameVaueCollection appSettings)` seems more readable)